src/yang/ietf-tcp.yang

```diff
@@ -270,6 +270,58 @@ module ietf-tcp {
     }
   } // grouping tcp-ecn-grouping
 
+  // congestion control
+
+  identity congestion-control-algorithm {
+    description
+      "Base identity from which all congestion control algorithms
+       are derived.";
+  }
+
+  identity Reno {
```

Comment:
Would suggest that we use a consistent scheme for naming attributes, and follow YANG style which mimics Lisp. No capital letters, and instead of camel case would use dash (-) to separate names.

In this case, 'Reno' should be 'reno'.